base/bconfig/outputbufferconfig.go

```diff
@@ -1,7 +1,10 @@
 package bconfig
 
 // OutputBufferConfig configures buffer and output settings
+
 type OutputBufferConfig struct {
-	BufferConfig ChunkBufferConfig // Verified buffer config
-	OutputConfig LogOutputConfig   // Verified output config
+	BaseConfig
```

Comment:
`BaseConfig` is probably not needed since OutputBufferConfig is a concrete struct and has no subtypes/implementations.